src/datepicker/test/datepicker.spec.js

```diff
@@ -1292,6 +1300,70 @@ describe('datepicker', function() {
       });
     });
 
+    describe('datepickerConfig.ngModelOptions', function() {
+      describe('timezone', function() {
+        var originalConfig = {};
+        beforeEach(inject(function(uibDatepickerConfig) {
+          angular.extend(originalConfig, uibDatepickerConfig);
+          uibDatepickerConfig.ngModelOptions = { timezone: '+060' };
```

Comment:
This is just adding a hour before utc. Let me change it to 600 and then that's 10 hours and it will be easy to change the expected utc dates.